src/participant/prompts/exportToPlayground.ts

```diff
@@ -0,0 +1,31 @@
+import * as vscode from 'vscode';
+
+export class ExportToPlaygroundPrompt {
+  static getAssistantPrompt(): vscode.LanguageModelChatMessage {
+    const prompt = `You are a MongoDB expert.
+Your task is to help the user build MongoDB queries and aggregation pipelines that perform their task.
+You convert user's code written in any programming language to the MongoDB Shell syntax.
+Take a user promt as an input string and translate it to the MongoDB Shell language.
+Keep your response concise.
+You should suggest queries that are performant and correct.
+Respond with markdown, suggest code in a Markdown code block that begins with \`\`\`javascript and ends with \`\`\`.
+Respond in MongoDB shell syntax using the \`\`\`javascript code block syntax.`;
+
+    // eslint-disable-next-line new-cap
+    return vscode.LanguageModelChatMessage.Assistant(prompt);
+  }
+
+  static getUserPrompt(prompt: string): vscode.LanguageModelChatMessage {
+    // eslint-disable-next-line new-cap
+    return vscode.LanguageModelChatMessage.User(prompt);
```

Comment:
Would it help to include any additional information about the user's code here?
We could have it wrapped in a user prompt, for instance:

`
Translate the following code from X language to MongoDB shell syntax:
```
...code
```
`

We can get file extension/language as well as the file name. I'm thinking the file name probably wouldn't help but I could see the language possibly being useful. Not sure.

Reply:
Ir seems working well without extra context. Or have you encountered some cases when it was returning unexpected results?

Reply:
Not yet no! Just raising the idea. We can iterate on it in future too if we find it's helpful.